CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ## [Unreleased]
+
+### Cast
+
+#### Added
+
+- Support for custom networks - accounts created on custom networks are saved in `accounts-file` under network's chain_id
+
+#### Removed
+
+- `--network` option together with the `network` field in Scarb.toml profile
```

Comment:
Shouldn't we deprecate it and then remove it? I wonder if we are scared of breaking changes or not 🤔

Reply:
It's early enough I'd say 👀

Reply:
I'd say let's just elaborate more in changelog, so it is clear why this change was introduced, and yolo it